app/controllers/LdapController.java

```diff
@@ -82,7 +82,7 @@ public Result apiTestLdapConnection() {
             log.error("Unable to connect", e);
             return internalServerError();
         }
-        return ok(Json.toJson(result));
+        return ok(Json.toJsonString(result)).as("application/json");
```

Comment:
Can we use a constant (e. g. [`MediaType#JSON_UTF_8`](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/net/MediaType.html#JSON_UTF_8) instead?